src/main/fc/tasks.c

```diff
@@ -336,7 +336,7 @@ void tasksInit(void)
 #endif
 
 #ifdef USE_PINIOBOX
-    setTaskEnabled(TASK_PINIOBOX, true);
+    pinioBoxTaskControl();
```

Comment:
Not sure this is still needed - the pinio task will be configured as needed every time the mode activation conditions are updated.
On the converse, running `pinioBoxTaskControl()` without first calculating the mode activation conditions might result in an unwanted (albeit harmless) outcome.